core/src/identity/auth/oauth/oauth.controller.ts

```diff
@@ -54,9 +56,10 @@ export class OauthController {
     @UseGuards(JwtRefreshGuard)
     @Get("refresh")
     async refreshTokens(@Request() req: Req): Promise<AccessToken> {
```

Comment:
How is the token actually refreshed? I.e., when/where is this `/refresh` endpoint called from the client? Shouldn't every request the client makes be able to refresh the token if it is expired?

Reply:
https://github.com/SprocketBot/sprocket/blob/8a89082050029cef3bce6217ff47e416352cc426/clients/web/src/hooks.ts#L31 is where it's called right now, from hooks.ts

Reply:
I have seen solutions that use interceptors with Axios to do what you're describing (refresh on any request) as well. This solution just uses what we currently have.